### PR TITLE
[AR] week 4 translation 

### PR DESCRIPTION
**Pull request by Ahmed-5**

### CODE REVIEW

File: docs/ar/week04/04-1.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
+---
+lang-ref: ch.04-1
+lecturer: Alfredo Canziani
+title: Linear Algebra and Convolutions
```
Comment (Atcold) [inline docs/ar/week04/04-1.md:4]: The title needs a translation.

File: docs/ar/week04/04-1.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+authors: Yuchi Ge, Anshan He, Shuting Gu, and Weiyang Wen
+date: 18 Feb 2020
+lang: ar
+translation-date: 22 Aug 2020
+translator: Ahmed Alhassan
+---
+
+## [Linear Algebra review](https://www.youtube.com/watch?v=OrBEon3VlQg&t=68s)
```
Comment (Atcold) [inline docs/ar/week04/04-1.md:12]: The title needs a translation.

File: docs/ar/week04/04-1.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+x_2 \begin{matrix} \rvert \\ \boldsymbol{a}_2 \\ \rvert \end{matrix} +
+    \cdots +
+x_n \begin{matrix} \rvert \\ \boldsymbol{a}_n \\ \rvert \end{matrix}
+$$
+
+<!-- The output is the weighted sum of the columns of matrix $\boldsymbol{A}$. Therefore, the signal is nothing but a composition of the input. -->
+
+المخرج عبارة عن مجموع بالأوزان لأعمدة المصفوفة $\boldsymbol{A}$. و هكذا, الإسترة ليست إلا تركيبة للمدخل.
+
+## [Extend Linear Algebra to convolutions](https://www.youtube.com/watch?v=OrBEon3VlQg&t=1030s)
```
Comment (Atcold) [inline docs/ar/week04/04-1.md:160]: The title needs a translation.

File: docs/ar/week04/04.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+lang-ref: ch.04
+title: Week 4
```
Comment (Atcold) [inline docs/ar/week04/04.md:3]: The title needs a translation.

Comment (Ahmed-5) [reply docs/ar/week04/04.md]: I translated the titles of both files and parts that I left untranslated in 04-1.md.
please check them and the config file and tell me if there's something wrong.

**Comment by Ahmed-5** [comment]
Hi,
Alfredo

I hope you are doing well. You know that arabic is written from right to
left so when I translated the text I searched for a solution for that and I
found it. I will modify the files again. Sorry for making you go through
this again and this will be the last time.

Best regards.
Ahmed

**Comment by Atcold** [comment]
Sure, no worries.
Let's open an issue, here on GitHub, so that we can find a solution for the whole translation group.
I'm suggesting an edit to the `HTML` template for the Arabic language. In this way all pages will inherit the modification.

**Comment by Ahmed-5** [comment]
I have found a way around this. Starting each paragraph with the unicode
&#x202b with wrapping each math formula with an extra curly braces so it
would not become glitchy solved the problem for me.

**Comment by Atcold** [comment]
I think we should be doing this globally, so that you don't have to do it manually.
Let's talk over an issue?

**Comment by Ahmed-5** [comment]
An issue?
Can you elaborate more on this?

**Comment by Atcold** [comment]
Can you open a GitHub issue and explain what we should do to *all* Arabic pages in order to visualise them properly?

**Comment by Ahmed-5** [comment]
Done

**Pull request merged**